app/views/organisers/problematic_projects.slim

```diff
@@ -0,0 +1,35 @@
+.page-header
+  h2.text-center.green-text Problematic Projects
+
+= link_to "Back to All projects", dashboard_organisers_path
+.row
+  .col-xs-12
+    table.table.table-striped.table-bordered
+      thead
+        tr
+          th.col-md-4 Mentee
+          th.col-md-4 Mentor
+          th.col-md-2 Project
+          th.col-md-2 GitHub Link
+          th.col-md-2 Missing
+      tbody
+        - @projects.each do |project| 
+          - if (project.mentee.is_missing == true) or (project.mentor.is_missing == true)
```

Comment:
@pg20 please create scope in a model and filter projects in controller - in view we'll simply display them